napari/_commands/_loader.py

```diff
@@ -0,0 +1,412 @@
+            The number of levels of the data.
+        """
+        if isinstance(data, list):
+            return len(data)
+        return 1
+
+    def _add_row(self, index: int, layer: Layer) -> int:
+        """Add row describing one layer.
+
+        Parameters
+        ----------
+        layer_id : int
+            The layer id (from view.cmd.layers).
+        layer : Layer
+            The layer itself.
+        """
+        layer_type = type(layer).__name__
+        num_levels = self._get_num_levels(layer.data)
+        shape_str = self._get_shape_str(layer)
+
+        # Use InfoDisplayer to display LayerInfo
+        info = chunk_loader.get_info(id(layer))
+        disp = InfoDisplayer(info) if info is not None else NoInfoDisplayer()
+
+        self.table.add_row(
+            [
+                index,
+                disp.sync,
+                disp.load_str,
+                layer.name,
+                layer_type,
+                disp.data_type,
+                num_levels,
+                disp.num_loads,
+                disp.num_chunks,
+                disp.total,
+                disp.avg_ms,
+                disp.mbits,
+                shape_str,
+            ]
+        )
+
+    def print(self):
+        """Print the whole table."""
+        self.table.print()
+
+
+class LevelsTable:
+    """Table showing the levels in a single layer.
+
+    Parameters
+    ----------
+    layer_id : int
+        The ID of this layer.
+    layer : Layer
+        Show the levels of this layer.
+    """
+
+    def __init__(self, layer):
+        self.layer = layer
+        self.table = RowTable(["LEVEL", "SHAPE", "TOTAL"])
+        self.table = RowTable(
+            ["LEVEL", {"name": "SHAPE", "align": "left"}, "TOTAL"]
+        )
+        data = layer.data
+        if isinstance(data, list):
+            for i, level in enumerate(data):
+                shape_str = level.shape if level.shape else "NONE"
+                size_str = naturalsize(level.nbytes, gnu=True)
+                self.table.add_row([i, shape_str, size_str])
+
+    def print(self):
+        """Print the whole table."""
+
+        self.table.print()
+
+
+class LoaderCommands:
+    """Layer related commands for the CommandProcessor.
+
+    Parameters
+    ----------
+    layerlist : List[Layer]
+        The current list of layers.
+    """
+
+    def __init__(self, layerlist: List[Layer]):
+        self.layerlist = layerlist
+
+    def __repr__(self):
+        return HELP_STR
+
+    @property
+    def help(self):
+        print(HELP_STR)
+
+    @property
+    def config(self):
+        """Print the current list of layers."""
+        src = async_config
+        config = [
+            ('synchronous', src.synchronous),
+            ('num_workers', src.num_workers),
+            ('log_path', src.log_path),
+            ('use_processes', src.use_processes),
+            ('delay_seconds', src.delay_seconds),
+            ('load_seconds', src.load_seconds),
+        ]
```

Comment:
```suggestion
        config = list(async_config.data.items())
```

this also echoes what I said about the AsyncConfig object being a little odd.